libsolidity/formal/Predicate.cpp

```diff
@@ -638,11 +638,13 @@ bool Predicate::fillArray(smtutil::Expression const& _expr, vector<string>& _arr
 
 map<string, optional<string>> Predicate::readTxVars(smtutil::Expression const& _tx) const
 {
+	// TODO: Remove `block.difficulty` in 0.9.0
```

Comment:
Do we have an issue for this filed for the 0.9.0 milestone yet? Just that we don't forget. We can also do it on the ``breaking`` branch right after the release already.

Reply:
No, I don't think so. I will add an issue for that.